CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Unreleased
 
+## New features
+
+### Replace back link placeholder URLs with javascript
+
+Previously, users had to manually replace the `href` for each Back link in the page templates.
```

Comment:
I'm not sure "Back link" should be capitalised. We should be consistent with the [design system guidance](https://design-system.service.gov.uk/components/back-link/)

Reply:
Ah, thanks!